index.html

```diff
@@ -1,75 +1,73 @@
 <!doctype html>
 <html lang="en-GB">
 <head>
-	<title>Home page</title>
-	<link rel="stylesheet" href="style.css" type="text/css" />
-	<link rel="stylesheet" href="navbar.css" type="text/css" />
-	<link rel="stylesheet" href="normalze.css" type="text/css" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="preconnect" href="https://fonts.gstatic.com">
-	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
-	<script src="https://kit.fontawesome.com/a5bc471788.js" crossorigin="anonymous"></script>
-	
+<title>Home page</title>
+<link rel="stylesheet" href="style.css" type="text/css" />
+<link rel="stylesheet" href="navbar.css" type="text/css" />
+<link rel="stylesheet" href="normalize.css" type="text/css" />
```

Comment:
This stylesheet needs to be loaded first to prevent it from overriding your other stylesheets.